src/components/SearchPlayers.js

```diff
@@ -0,0 +1,94 @@
+//@flow strict
+
+import * as React from 'react';
+
+type Props = {|
+	playerList: Array<string>,
+|};
+
+type State = {|
+	searchName: string,
+	searchList: Array<string>,
+	displayList: React.Node
+|};
+
+// Search Players Component. Takes an array of all players.
+//
+// Prop Names:
+// playerList (Array of all users)
+// 
+// EXAMPLE PROP USAGE = <SearchPlayers playerList={ ["jeffery", "jeff", "john", "illeatbutt", "childeatbigtime", "childconsumer"] }/>
+//
+// Needs to be updated to have a challenge player button.
+class SearchPlayers extends React.Component<Props, State> {
+
+	constructor(props: Props) {
+		super(props);
+
+		this.state = {
+			searchName: "",
+			searchList: [""],
+			displayList: <div className="searchPlayersList"></div>
+		}
+	}
+
+	// When typing in the searchbar, update searchName.
+	handleChange = (e: string) => {

```

Comment:
I mention how you should change this later, but you need to specify the return type too